src/App.vue

```diff
@@ -178,4 +180,8 @@ blockquote.blockquote {
   border-left: 2px solid lightgray;
 }
 
+html, body {
+  font-family: 'Poppins', sans-serif;
```

Comment:
@iimpulse I can't remember if we said 'let's add Poppins font to the whole page.' I thought we were discussing only a menu item. I'll assign that to jet lag. I'm sure we'll hear about this one too. :bowtie: